components/extensions/vscode/src/Commands.ts

```diff
@@ -19,12 +19,15 @@
 import * as path from "path";
 import * as vscode from "vscode";
 import Constants from "./constants";
+import CommandsUtils from "./utils/CommandsUtils";
 
 /**
  * Cellery commands.
  */
 class Commands {
     private static readonly terminals: { [name: string]: vscode.Terminal } = { };
```

Comment:
Shall we remove this and directly check if the terminal is present and get it using the VS Code API ?